frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/storage-tab-initial-state.ts

```diff
@@ -71,15 +70,15 @@ const getUrlStorage = (storageClassConfigMap: ConfigMapKind) => ({
   volume: VolumeWrapper.initializeFromSimpleData({
     name: ROOT_DISK_NAME,
     type: VolumeType.DATA_VOLUME,
-    typeData: { name: prefixedID(VM_TEMPLATE_NAME_PARAMETER, ROOT_DISK_NAME) },
+    typeData: { name: generateDataVolumeName(vmName, ROOT_DISK_NAME) },
```

Comment:
again I know we override it, but it is nice to have it clean for debugging purposes. Can you please create a variable so the volume references the right DV?

Reply:
Excellent point. Done!